src/main/java/com/example/monitoring/core/email/EmailContext.java

```diff
@@ -0,0 +1,9 @@
+package com.example.monitoring.core.email;
+
+public enum EmailContext {
+    WELCOME_MESSAGE,
+    ALERT,
+    PASSWORD_RESET,
+    DELETE_ACCOUNT
```

Comment:
We shouldn't delete accounts. We should strip the user of access rights but keep him in the database for documentation reasons. In case we need to grant access we are just changing access rights, not creating a new user. Moreover, do we want to notify user of deletion? Why? We can notify that their access rights were changed.

Reply:
It looks like we currently want only one template to use - ALERT. I think it's quite unreasonable to delete that kind of option for future improvements, so I'm going to delete all but ALERT. Besides that I will add TEST as EmailContext just to leave that kind of structure. Is that okay?

Reply:
So... about this access rights change, I forgot to mention - should I add ACCESS_RIGHTS_CHANGE context?

Reply:
Yes add ACCESS_RIGHTS_CHANGE and rename WELCOME_MESSAGE to TEST if it is meant for test puproses. Others can be deleted. We can leave DELETE_ACCOUNT for possible future feature but I doubt we will use it